common/models/user.js

```diff
@@ -361,11 +360,14 @@ module.exports = function(User) {
     assert(typeof options === 'object', 'options required when calling user.verify()');
     assert(options.type, 'You must supply a verification type (options.type)');
     assert(options.type === 'email', 'Unsupported verification type');
-    assert(options.to || this.email, 'Must include options.to when calling user.verify() or the user must have an email property');
+    assert(options.to || this.email,
+      'Must include options.to when calling user.verify() ' +
+      'or the user must have an email property');
     assert(options.from, 'Must include options.from when calling user.verify()');
 
     options.redirect = options.redirect || '/';
-    options.template = path.resolve(options.template || path.join(__dirname, '..', '..', 'templates', 'verify.ejs'));
+    var defaultTemplate = path.join(__dirname, '..', '..', 'templates', 'verify.ejs');
```

Comment:
Possibly not on windows. However, let's keep this PR focused on `eslint `issues, I don't want to introduce any other changes.